src/distributing-policies/secure-supply-chain.md

```diff
@@ -0,0 +1,567 @@
+so that the policy can be trusted.
+
+Each of these sections can contain one or more signature requirements.
+Users can also define both sections in one file as shown above. In this situations, all
+the signatures requirements from the `allOf` **AND** a minimum number of matches
+from the `anyOf` section as described in the `minimumMatches` field must be valid.
+
+### Signature validation
+
+The users can validate different signatures in the `anyOf` and `allOf` sections.
+It's possible to validate the public key and the keyless data used to sign the policy.
+
+### Public key validation
+
+To check if the policy is assigned with the given public key, the users can
+define the key data and the owner of the key used to sign the policy. As illustrated
+bellow, it is necessary to define the kind as `pubKey` and insert the public
+key data in the `key` field. The owner field is optional.
+
+**Example**
+
+```  yaml
+  - kind: pubKey
+    owner: bob # optional
+    key: |
+      -----BEGIN PUBLIC KEY-----
+      MBFKHFDGHKIJH0CAQYIKoZIzj0DAQcDQgAEX0HFTtCfTtPmkx5p1RbDE6HJSGAVD
+      BVDF6SKFSF87AASUspkQsN3FO4iyWodCy5j3o0CdIJD/KJHDJFHDFIu6sA==
+      -----END PUBLIC KEY-----
+```
+
+### Keyless signatures validation
+
+When signed in keyless mode, we do not have the public key to verify. In this
+situation, the users can verify the field filled with the OIDC data used during
+the signing process. For that, it's necessary to define the kind of the signature
+validation as `genericIssuer`. Therefore, it's possible to verify three pieces
+of information from the signature:
+
+- `issuer`(mandatory): matches the `Issuer` attribute in the certificate
+generated by Fulcio. This shows the OIDC used to sign the policy
+- `subject`: field used to match the `Subject` attribute in Fulcio's
+certificate. The `Subject` field contains the information of the user used to
+authenticate again the OIDC provider. The verification field, `subject`, can have
+two children fields: `equal` forces the `Subject` from the certificate to be
+equal to the value defined in the signature validation; `urlPrefix` forces the
+value of the certificate's `Subject` field value to be prefixed by the value
+defined in the signature requirement.
+
+**Examples**
+
+The following signature requires the policy to be signed by Alice in a keyless manner
+using the GitHub OIDC:
+
+```yaml
+- kind: genericIssuer
+  issuer: https://github.com/login/oauth
+  subject:
+    equal: alice@example.com
+```
+
+This signature requirement enforces that the policy be signed in the context of a GitHub actions environment
+from a repository owned by Flavio:
+
+```yaml
+- kind: genericIssuer
+  issuer: https://token.actions.githubusercontent.com
+  subject:
+    urlPrefix: https://github.com/flavio
+```
+
+### GitHub actions signature verification
+
+The signature validation kind, `githubAction` is used to validate policies signed in a Github
```

Comment:
```
The signature validation kind, `githubAction` is used to validate policies signed in a GitHub
```

Reply:
Sorry, I did not get it. What should I change?

Reply:
sorry, I didn't tag the snippet a `suggestion`. It was a silly typo about GitHub name